manager/integration/tests/test_scheduling.py

```diff
@@ -794,6 +787,9 @@ def test_data_locality_basic(client, core_api, volume_name, pod, settings_reset)
     assert volume3.replicas[0]["mode"] == "RW"
     assert volume3.replicas[0]["running"] is True
 
+    delete_and_wait_pod(core_api, pod3_name)
+    wait_for_volume_detached(client, volume3_name)
+
```

Comment:
Make sure to clean up the pod after we done with it. Otherwise, PVC cannot be removed at the in the finalized at the end of the test.